docs/sources/administration/plugin-management/_index.md

```diff
@@ -78,7 +78,7 @@ Before following the steps below, make sure you are logged in as a Grafana admin
 <a id="#plugin-catalog-entry"></a>
 Currently, there are two entry points to the Plugin catalog.
 
-- Grafana server administrators can find it at **Server Admin >
+- Grafana server administrators can find it at **Configuration >
   Plugins**.
 - Organization administrators can find it at **Configuration > Plugins**.
 
```

Comment:
There's a bit of duplication here - I think we can rewrite these lines to something like:

`Administrators can find the Plugin catalog at **Configuration > Plugins**`